tools/validators/instance_validator/generate_universe.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generates the ontology universe for the instance validator."""
+
+from __future__ import print_function
+
+from sys import path
+import os
+
+# add universe building packages to path
+path.append(os.path.abspath(os.path.join('..', 'ontology_validator')))
+# add ontology files to path
+path.append(os.path.exists(os.path.join('..', '..', '..', 'ontology')))
+
+from yamlformat.validator import external_file_lib
+from yamlformat.validator import presubmit_validate_types_lib
+
+def build_universe():
+  """Generates the ontology universe.
+
+  Returns:
+    Generated universe object.
+  """
+
+  ontology_validator_exists = os.path.exists(os.path.join(
+      '..', 'ontology_validator'))
```

Comment:
I think this was a design decision @charbull recommended? Maybe the reasoning was to prevent a possible failure in the script from throwing an ugly stacktrace error.

Reply:
I think it is good to keep it just in case we moved stuff in the future or someone pulled it and tried to run the instance validator